<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="arduinobot">

  <ros2_control name="RobotSystem" type="system">
    <hardware>
      <plugin>gazebo_ros2_control/GazeboSystem</plugin>
    </hardware>

    <joint name="joint_1">
      <command_interface name="position"/>
      <state_interface name="position"/>
    </joint>

    <joint name="joint_2">
      <command_interface name="position"/>
      <state_interface name="position"/>
    </joint>

    <joint name="joint_3">
      <command_interface name="position"/>
      <state_interface name="position"/>
    </joint>

    <joint name="joint_4">
      <command_interface name="position"/>
      <state_interface name="position"/>
    </joint>
  </ros2_control>

</robot>
